### DISCUSSION TITLE
Add HTML element classes like smooth-scroll, scroll-padding etc.

### DISCUSSION THREAD

**Discussion by jarihant701** [tailwindlabs/tailwindcss#5072] [Ideas]
I think there are some utility classes that can be added like `scroll-behaviour: smooth` (if it is enabled by default it would look good).
Adding classes for scroll padding and many more such scroll utilities.

I can try adding these if you can assign this to me.

**Comment by lukewarlow** [comment]
A PR for adding scroll behaviour has now been merged so should be in Tailwind 3
